libp2p/tools/utils.py

```diff
@@ -58,24 +57,6 @@
     return tuple(nodes_list)
```

Comment:
Should the type here also be `RoutedHost`?

Reply:
This function is only used once and does not ease any cognitive load. Let me just remove it.